pgx/src/layout.rs

```diff
@@ -71,7 +71,8 @@ pub(crate) enum Align {
     Byte = mem::align_of::<u8>(),
     Short = mem::align_of::<libc::c_short>(),
     Int = mem::align_of::<libc::c_int>(),
-    Double = mem::align_of::<f64>(),
+    // 32-bit processors have a maximum alignment of 32 bits
+    Double = mem::align_of::<libc::c_int>() * 2,
```

Comment:
This change is incorrect, this variant's value is in fact intended to match the alignment of Double and the name was not idly selected. If this causes a compiler failure due to reusing the same number for two different variants, please refactor the appropriate code that exploits the `this_enum as usize` casts.

Reply:
It turns out pgx never relied on the variant's value anywhere, so I just removed the variant values. `Align` isn't publicly exported so user code is unaffected.

Reply:
Oh! Well, that's a completely fair solution.